### ISSUE TITLE
Need windows build instructions

### ISSUE THREAD

**Issue by dts350z** [phip1611/audio-visualizer#5]
I don't see any build instructions?

Especially for windows 10?

**Comment by phip1611** [comment]
Hey! I'm currently at RustConf and back in Germany at the end of August
(vacation). I can help you no earlier than that, unfortunately. Thanks for
using one of my crates, tho! I hope, you either can figure it out yourself
until then or that you can wait a few weeks.

Kind regards
Philipp

dts350z ***@***.***> schrieb am Fr., 5. Aug. 2022, 09:25:

**Comment by phip1611** [comment]
hey @dts350z , it builds without problems on my Windows 10 machine. Which problems/error messages do you encounter?

**Comment by dts350z** [comment]
Sorry, I thought it was a library written in some compiled language like C or C++. I see now that it is rust. I installed rust on Windows but still don't know how to use your library, or even run the examples.

**Comment by phip1611** [comment]
Ahh, so you do not have a Rust background. I see. So usually, because Rust has a unified build system, projects do not have individual build instructions as the "typical instructions" just work.

TL;DR: `cargo run --example <name-of-example>` will do the trick

You do not need to build the library to include it in another project as the default for Rust is to use static linkage. Just create a Rust project with Cargo, add this crate as dependency, and then you should be able to use and compile everything.
